src/Symfony/Component/Serializer/Tests/SerializerTest.php

```diff
@@ -714,6 +779,38 @@ public function __construct($value)
     }
 }
 
+class Baz
+{
+    public $list;
+
+    public $settings = [];
+
+    public function __construct(array $list)
+    {
+        $this->list = new UserList($list);
+    }
+}
+
+class UserList implements \Countable, \IteratorAggregate
```

Comment:
`DummyList` or smth to avoid confusion :smile:

Reply:
I'm not sure why it's better, but I updated it